payments/migrations/0004_basket_integrated_system_alter_basket_user.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 4.2.16 on 2024-10-16 18:16
+
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ("system_meta", "0005_integratedsystem_payment_process_redirect_url"),
+        ("payments", "0003_alter_order_state"),
+    ]
+
+    operations = [
+        migrations.AddField(
```

Comment:
Needs an op here to clear out existing baskets before adding the field. (I got an integrity error migrating because my local system had baskets in place.)